lang/package.go

```diff
@@ -17,6 +17,11 @@ import (
 	"github.com/cloudogu/gomarkdoc/logger"
 )
 
+var (
+	actualPackage string
```

Comment:
Quite too often, the term `current` does fit better than `actual` in the case of an inspection in a point of time (which may change during earlier or later). In contrast `actual` describes something as what something in reality is or as a matter of fact.

```suggestion
	currentPackage string
```